chanzuckerberg/idseq-web
# Fix bug where date to become public was not displayed correctly.

### Description

Fix issue where date when next sample will become public was incorrect on `ProjectSettingsModal`.

### Tests

* Open a private project > Settings
* Verify that date on top header of the modal is correct.
